MoPubSDK/Native Ads/MPCollectionViewAdPlacer.m

```diff
@@ -117,6 +123,19 @@ - (void)adPlacer:(MPStreamAdPlacer *)adPlacer didLoadAdAtIndexPath:(NSIndexPath
 
 - (void)adPlacer:(MPStreamAdPlacer *)adPlacer didRemoveAdsAtIndexPaths:(NSArray *)indexPaths
 {
+    for (const id obj in indexPaths) {
+        if (![obj isKindOfClass:[NSIndexPath class]]) {
+            return;
```

Comment:
should this be `continue` to process the rest of the array versus returning early?